app/workers/commit_monitor_handlers/commit_range/github_pr_commenter/commit_message_username_checker.rb

```diff
@@ -0,0 +1,62 @@
+require 'rugged'
+
+module CommitMonitorHandlers::CommitRange
+  class GithubPrCommenter::CommitMessageUsernameChecker
```

Comment:
Should this be a more generic `CommitMetadataChecker`?  I could see wanting to check the commit message and other things too

Reply:
Sure, I can change it to that.

Reply:
Updated.